streams/test-utils/src/main/java/org/apache/kafka/streams/test/ConsumerRecordFactory.java

```diff
@@ -175,11 +180,28 @@ public void advanceTimeMs(final long advanceMs) {
             -1L,
             timestampMs,
             TimestampType.CREATE_TIME,
-            ConsumerRecord.NULL_CHECKSUM,
+            (long) ConsumerRecord.NULL_CHECKSUM,
             serializedKey == null ? 0 : serializedKey.length,
             serializedValue == null ? 0 : serializedValue.length,
             serializedKey,
-            serializedValue);
+            serializedValue,
+            headers);
+    }
+
+    /**
+     * Create a {@link ConsumerRecord} with default topic name and given topic, key, value, and timestamp.
```

Comment:
remove `default topic name` -- this method takes `topicName` parameter

Reply:
ack.